btcjson/chainsvrresults.go

```diff
@@ -91,6 +91,26 @@ type GetBlockVerboseResult struct {
 	NextHash      string        `json:"nextblockhash,omitempty"`
 }
 
+// GetBlockVerboseResultReply
+type GetBlockVerboseResultReply struct {
```

Comment:
As indicated in [this comment](https://github.com/btcsuite/btcd/issues/1653#issuecomment-710673166), I prefer reusing the existing `GetBlockVerboseResult` and populate the `Tx` field, since it's desirable to be compatible with Bitcoin Core.

You'll need to update the type of `Tx` field in the original `GetBlockVerboseResult` like this:

```go
Tx            []TxRawResult `json:"tx,omitempty"` 
```